qemu/tests/cfg/qemu_img_negative.cfg

```
@@ -26,3 +26,14 @@
                     image_name_new = "images/rebase_negative_new"
                     create_image_new = no
                     rebase_list = "sn > new"
+        - luks_with_non_utf8_secret:
+            only luks
+            type = image_creation_luks_with_non_utf8_secret
+            start_vm = no
+            create_image = no
+            images = stg
+            err_info = "Data from secret\s+\w+\s+is not valid UTF-8"
+            non_utf8_secret = "echo -n -e '\x3a\x3c\x3b\xff' > %s"
```

```suggestion
            echo_non_utf8_secret_cmd = "echo -n -e '\x3a\x3c\x3b\xff' > %s"
```
maybe more readable.

---

updated, thanks.